ui/static/ui/css/slide-drawer.css

```diff
@@ -81,6 +81,7 @@ html, body {
   -webkit-transition: visibility 0s 0s;
   -moz-transition: visibility 0s 0s;
   transition: visibility 0s 0s;
+  overflow: scroll;
```

Comment:
This is for handling scroll overlap between panels and background. Before this fix you can vertically scroll panel and background same time, which case issue on animate scroll up in case of error message or success.

Now at the time one scroll will be enable i.e if you open a panel i will disable background page scroll